src/jcsclient/help_topics/rds/create-db-instance.txt

```diff
@@ -65,10 +67,13 @@ Specify the daily time range during which JCS can run the automated backup tasks
 Specify the number of days that JCS RDS must retain the automated database backups. JCS RDS retains the automated DB snapshot backups till the specified period. Supported Range: 0 -35 (days).
 
 
-        
+    [--log-retention-period <integer>]
+Specify the number of days for which  JCS RDS must retain specific log files of the instance. Supported Range: 1 -7 (days).
```

Comment:
Change it to -

Specify the number of days for which  JCS RDS must retain specific log files of the instance. Supported Range: 1-7 (days). Default: 1 day.